internal/fingerprint/fingerprint_test.go

```diff
@@ -389,7 +391,7 @@ func TestInMemFingerprintingCompressedContent(t *testing.T) {
 		{
 			name:        "Nupkg",
 			path:        "testdata/archive/nupkg",
-			expected:    22,
+			expected:    21,
```

Comment:
We incorrectly fingerprinted newtonsoft.json.13.0.3.nupkg/[Content_Types].xml before

Reply:
good find!